transforms/csdl/preprocess_csdl.xsl

```diff
@@ -501,4 +501,62 @@
       </xsl:element>
     </xsl:copy>
   </xsl:template>
+  <xsl:template match="edm:Schema[@Namespace='microsoft.graph']">
```

Comment:
Hey @baywet @irvinesunday.
It looks like this change is causing some tests to break on the SDK generator.

Should the match be applied indiscriminately? Or should it be more specific as seen [here](https://github.com/microsoftgraph/msgraph-metadata/blob/61269bb0c736df1276e66593173438efc2162d3a/transforms/csdl/preprocess_csdl.xsl#L469). 

The precondition assumed in the [test](https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/blob/dev/test/Typewriter.Test/Given_a_valid_metadata_file_to_Typewriter.cs#L560) will no longer apply as the cleanMetadata will have some capability annotations. This is irrespective of whether an input CSDL has the target present.

Reply:
I'm not sure, to me adding a few global annotations should not impact the overall generation besides the targeted items.
That's unless some closing tag is missing/out of order...because you've got to love CSDL...